### ISSUE TITLE
Facebook includes every frame in a whole album instead of displaying an individual post per frame

### ISSUE THREAD

**Issue by Apo458** [TissuePowder/YAFOT-Facebook#9]
![image_2022-10-08_135555159](https://user-images.githubusercontent.com/65506716/194725699-978ee67d-9506-4ed5-9bdf-e83088f5395b.png)
It shows an entire album instead of showing a dedicated post for an individual frame

**Comment by TissuePowder** [comment]
show me the command you are using

**Comment by Apo458** [comment]
Turns out I grabbed the ID that the URL displays when you enter the page. I headed to the "Photos" section and clicked on a random image, I noticed that there was another ID aside from the main one. After replacing the ID on the command, the issue got fixed and now the frames are shown properly:
![image](https://user-images.githubusercontent.com/65506716/194771523-0e19cc63-100e-4bb4-8b01-3ba2aa372bf0.png)

If someone happens to find this thread and you're dealing with the same issue, now you know how to fix it. 

Sorry for the inconvenience @TissuePowder
